src/templates/header/index.hbs

```diff
@@ -42,9 +42,15 @@
                     {{> nav-alternate-language-links }}
 
                     <li class="c-nav-list-item c-nav-list-item--support" data-test-id="support">
-                        <a href="{{ helpUrl }}" data-gtm="header|click - navigation|help" class="c-nav-list-link">
-                            {{ i18n "helpLinkText" }}
-                        </a>
+                        {{#if isHelpExternal}}
```

Comment:
You can do `{{#if false}}target="_blank"{{/if}}` here. I've tested it out and it works fine.